packages/testing/playwright/composables/DataTablesComposer.ts

```diff
@@ -0,0 +1,37 @@
+import type { n8nPage } from '../pages/n8nPage';
+
+export class DataTableComposer {
+	constructor(private readonly n8n: n8nPage) {}
+
+	async createNewDataTable(name: string) {
+		const nameInput = this.n8n.dataTable.getNewDataTableNameInput();
+		await nameInput.fill(name);
+		await this.n8n.dataTable.getNewDataTableConfirmButton().click();
+	}
+
+	/**
+	 * Creates project and data table inside it, navigating to project 'Data Table' tab
+	 * @param projectName
+	 * @param dataTableName
+	 * @param source - from where the creation is initiated (empty state or header dropdown)
+	 */
+	async createDataTableInNewProject(
+		projectName: string,
+		dataTableName: string,
+		source: 'empty-state' | 'header-dropdown',
+	) {
+		await this.n8n.projectComposer.createProject(projectName);
```

Comment:
There's an API helper as well that could make this easier too
n8n.start.fromNewProject()
Although it ends up on the canvas not the home page.